src/style.css

```
@@ -0,0 +1,70 @@
+* {
+  margin: 0;
+}
+
+.blocks {
+  display: flex;
+  flex-wrap: wrap;
+  background: #000;
+}
+
+.blocks__block {
+  width: calc(100% / 3);
```

```suggestion
  width: calc(100% / 3);
```
Кмітливе рішення, але це задача з теми flexbox, тому її мета - допомогти краще розібратись із флексбоками.
Давай спробуємо використати не таке обчислення, а властивість flex-basis